resources/assets/js/components/Datatable.vue

```diff
@@ -55,7 +75,27 @@
                     this.getData()
                 },
                 500
-            )
+            ),
+            goPage(page, event) {
+                if (event) {
+                    event.preventDefault()
+                }
```

Comment:
You can add *prevent* to the *v-on* as `v-on:click.prevent` or `@click.prevent="changePage('next')"` to avoid this control